zendesk/zendesk_client_service.go

```diff
@@ -243,19 +243,32 @@ func (c *client) getAll(endpoint string, in interface{}) ([]Ticket, error) {
 	apiStartIndex := strings.Index(dataPerPage.NextPage, apiV2)
 	prevPage := ""
 
-	for dataPerPage.NextPage != prevPage {
-		if fieldName == "tickets" {
-			result = append(result, dataPerPage.Tickets...)
+	for {
```

Comment:
infinite loop scares me little bit. I know there's a break condition. But, is there a better way?

Reply:
sure. I can change that part